backend/src/common/utils/paginate.utils.ts

```diff
@@ -0,0 +1,25 @@
+import { PaginationDto } from '../dto/dto';
+
+export async function paginate<T>(
+  data: T,
+  total: number,
+  page: number,
+  limit: number,
+): Promise<PaginationDto<T>> {
+  const totalPages = Math.ceil(total / limit);
+  const currentPage = page;
+  const nextPage = page < totalPages ? page + 1 : null;
+  const prevPage = page > 1 ? page - 1 : null;
+
+  const paginationResponse = new PaginationDto<T>();
+  paginationResponse.data = data;
+  paginationResponse.meta = {
+    limit: limit,
+    total: total,
+    current_page: currentPage,
+    total_pages: totalPages,
+    next: nextPage,
+    prev: prevPage,
+  };
+  return paginationResponse;
```

Comment:
```suggestion
  return {
    items,
    meta: {
      limit: limit,
      total: total,
      current_page: page,
      total_pages: totalPages,
      next: nextPage,
      prev: prevPage,
    }
  };
```

[metaSchema](https://github.com/jiphyeonjeon-42/backend-nest/blob/feeb8f473d1220e64b36a80f5a6136bc365bcf3b/backend/src/common/dtos/meta.dto.ts#L6-L28)와 [PageType](https://github.com/jiphyeonjeon-42/backend-nest/blob/feeb8f473d1220e64b36a80f5a6136bc365bcf3b/backend/src/common/dtos/page.dto.ts#L4-L14)을 쓰는 것이 좋을 것 같습니다.